README.md

```diff
@@ -1 +1,24 @@
 # DigitalBank
+CASOS PRUEBA TOMADOS EN CUENTA:
```

Comment:
This doc should be in a different file than the readme. Consider creating a file for documenting test cases exclusively